.travis.yml

```diff
@@ -19,7 +19,9 @@ matrix:
         - php: hhvm
 
 before_install:
+    - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - composer self-update
+    - if [ "$GITHUB_OAUTH_TOKEN" ]; then composer config --global github-oauth.github.com ${GITHUB_OAUTH_TOKEN}; fi;
```

Comment:
I'm not sure, but compare the PR build (no token):
https://travis-ci.org/api-platform/core/jobs/114568574#L177

to the build on my fork (with token):
https://travis-ci.org/teohhanhui/DunglasApiBundle/jobs/114568523#L173